crates/extensions/tedge_mqtt_bridge/src/backoff.rs

```diff
@@ -78,7 +78,7 @@ mod tests {
         let now = Instant::now();
         let clock = IterClock::new([now, now + Duration::from_secs(1)]);
         let mut backoff = deterministic_backoff(&clock);
-        assert_eq!(backoff.backoff(), Duration::from_secs(30));
+        assert_eq!(backoff.backoff().as_secs(), 30);
```

Comment:
I've updated these tests following failures like this when running `cargo stress`:

```
failures:

---- backoff::tests::backoff_is_30_seconds_after_exceeding_timeout stdout ----
thread 'backoff::tests::backoff_is_30_seconds_after_exceeding_timeout' panicked at crates/extensions/tedge_mqtt_bridge/src/backoff.rs:124:9:
assertion `left == right` failed
  left: 30.000000001s
 right: 30s
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```